packages/theme/css/foundations/size.css

```diff
@@ -51,4 +47,5 @@
 .salt-density-low,
 .salt-density-touch {
   --salt-size-bar-small: 2px;
```

Comment:
Going with this pattern has introduced inconsistency, size-border is 1px everywhere but not in this list

Reply:
have reverted the change above to keep format in line